lib/rubocop/cop/rspec/multiple_expectations.rb

```diff
@@ -31,6 +31,26 @@ module RSpec
       #     end
       #   end
       #
+      # @example with aggregate_failures (defaults to true)
```

Comment:
I suggest:
```
@example `aggregate_failures: true` (default)
```

Reply:
👍🏾 Thanks for the context. I was trying to find a similar convention in the existing examples. I made the change. The only question was if it made sense to inject a `when` so that it it the rendered document heading shows up as:

when`aggregate_failures: true` (default) 

versus

`aggregate_failures: true` (default)
```
 @example when `aggregate_failures: true` (default)
```

Reply:
Skimming through existing cops' docs, it seems that we don't add "when", even though it improves readability. [Example](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#examples-16). I'd stick with the tradition.